tests/python_package_test/test_basic.py

```diff
@@ -592,7 +592,7 @@ def test_choose_param_value_objective(objective_alias):
 
 @pytest.mark.parametrize('collection', ['1d_np', '2d_np', 'pd_float', 'pd_str', '1d_list', '2d_list'])
 @pytest.mark.parametrize('dtype', [np.float32, np.float64])
-def test_list_to_1d_numpy(collection, dtype):
+def test__list_to_1d_numpy(collection, dtype):
```

Comment:
```suggestion
def test_list_to_1d_numpy(collection, dtype):
```

Thanks for the thoroughness, but please revert this line. It isn't necessary for the test name to exactly match the function name, and I'd prefer to leave this line unchanged so that the [git blame](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#viewing-the-line-by-line-revision-history-for-a-file) is refers to the last time the test definition line was materially changed.